offchain-modules/packages/scripts/src/devDocker.ts

```diff
@@ -0,0 +1,344 @@
+  ETH_PRIVATE_KEY: string,
+  CKB_PRIVATE_KEY: string,
+  password,
+) {
+  const baseConfig: Config = lodash.cloneDeep(initConfig);
+  logger.debug(`baseConfig: ${JSON.stringify(baseConfig, null, 2)}`);
+  baseConfig.eth.assetWhiteList = assetWhiteList;
+  baseConfig.eth.contractAddress = ethContractAddress;
+  baseConfig.eth.multiSignThreshold = multisigConfig.threshold;
+  baseConfig.eth.multiSignAddresses = multisigConfig.verifiers.map((v) => v.ethAddress);
+  baseConfig.ckb.deps = ckbDeps;
+  baseConfig.ckb.multisigScript = {
+    R: 0,
+    M: multisigConfig.threshold,
+    publicKeyHashes: multisigConfig.verifiers.map((v) => v.ckbPubkeyHash),
+  };
+  baseConfig.ckb.ownerCellTypescript = ownerCellConfig.ownerCellTypescript;
+  baseConfig.ckb.multisigLockscript = ownerCellConfig.multisigLockscript;
+  baseConfig.ckb.startBlockHeight = ckbStartHeight;
+  baseConfig.eth.startBlockHeight = ethStartHeight;
+  // collector
+  const collectorConfig: Config = lodash.cloneDeep(baseConfig);
+  collectorConfig.common.role = 'collector';
+  collectorConfig.common.orm.host = 'collector_db';
+  collectorConfig.eth.multiSignHosts = multisigConfig.verifiers.map((v, i) => {
+    return {
+      address: v.ethAddress,
+      host: `http://verifier${i + 1}/force-bridge/sign-server/api/v1`,
+    };
+  });
+  collectorConfig.ckb.multiSignHosts = multisigConfig.verifiers.map((v, i) => {
+    return {
+      address: v.ckbAddress,
+      host: `http://verifier${i + 1}/force-bridge/sign-server/api/v1`,
+    };
+  });
+  const collectorStore = KeyStore.createFromPairs(
+    {
+      ckb: CKB_PRIVATE_KEY,
+      eth: ETH_PRIVATE_KEY,
+    },
+    password,
+  ).getEncryptedData();
+  const collectorKeystorePath = path.join(configPath, 'collector/keystore.json');
+  writeJsonToFile(collectorStore, collectorKeystorePath);
+  writeJsonToFile({ forceBridge: collectorConfig }, path.join(configPath, 'collector/force_bridge.json'));
+  // watcher
+  const watcherConfig: Config = lodash.cloneDeep(baseConfig);
+  watcherConfig.common.role = 'watcher';
+  watcherConfig.common.orm.host = 'watcher_db';
+  const watcherStore = KeyStore.createFromPairs(
+    {
+      ckb: CKB_PRIVATE_KEY,
+      eth: ETH_PRIVATE_KEY,
+    },
+    password,
+  ).getEncryptedData();
+  const watcherKeystorePath = path.join(configPath, 'watcher/keystore.json');
+  writeJsonToFile(watcherStore, watcherKeystorePath);
+  writeJsonToFile({ forceBridge: watcherConfig }, path.join(configPath, 'watcher/force_bridge.json'));
+  // verifiers
+  multisigConfig.verifiers.map((v, i) => {
+    const verifierIndex = i + 1;
+    const verifierConfig: Config = lodash.cloneDeep(baseConfig);
+    verifierConfig.common.role = 'verifier';
+    verifierConfig.common.orm.host = `verifier${verifierIndex}_db`;
+    verifierConfig.eth.privateKey = 'verifier';
+    verifierConfig.ckb.privateKey = 'verifier';
+    verifierConfig.common.collectorPubKeyHash.push(privateKeyToCkbPubkeyHash(CKB_PRIVATE_KEY));
```

Comment:
collectorConfig also need collectorPubKeyHash to verify pendingTx from verifier

Reply:
fixed in another PR.